#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

geometry_msgs/Point32 point
float32 echo_width
uint8 device_id
uint8 layer
uint8 echo
duration timestamp
uint16 flags